modules/cli/src/main/scala/scala/cli/commands/HelpCmd.scala

```diff
@@ -0,0 +1,16 @@
+package scala.cli.commands
+
+import caseapp._
+
+import scala.cli.ScalaCli
+case class HelpOptions()
+
+object HelpOptions {
+  implicit lazy val parser: Parser[HelpOptions] = Parser.derive
+  implicit lazy val help: Help[HelpOptions]     = Help.derive
+}
+
+object HelpCmd extends ScalaCommand[HelpOptions] {
```

Comment:
Something like `override def names = Seq(Seq("help"))` might be needed here, for `scala-cli help` to work.